.goreleaser.yaml

```diff
@@ -0,0 +1,71 @@
+project_name: cadctl
+builds:
+  - ldflags:
+      - "-s -w"
+      - "-extldflags=-zrelro"
```

Comment:
you had comments before on zrelro being a requirement for stuff, can you add it again?

Reply:
Oh yeah.. I don't know where they went

Reply:
done